app/javascript/packages/rails-i18n-webpack-plugin/rails-i18n-webpack-plugin.js

```diff
@@ -27,7 +27,7 @@ const ExtractKeysWebpackPlugin = require('./extract-keys-webpack-plugin.js');
  * // 'foo'
  * ```
  *
- * @param {Record<string, any>} object
+ * @param {undefined|Record<string, any>} object
```

Comment:
does marking it as optional like this work here?
```suggestion
 * @param {Record<string, any>=} object
```

Reply:
It looks like TypeScript would treat that as an optional parameter and then get mad about a required parameter following an optional parameter.

```
app/javascript/packages/rails-i18n-webpack-plugin/rails-i18n-webpack-plugin.js:35:22 - error TS1016: A required parameter cannot follow an optional parameter.

35 function dig(object, keyPath) {
                        ~~~~~~~
```